alignak/objects/schedulingitem.py

```diff
@@ -1547,6 +1547,9 @@ def consume_result(self, chk, notif_period, hosts,  # pylint: disable=R0915,R091
         ok_up = self.__class__.ok_up  # OK for service, UP for host
 
         # ============ MANAGE THE CHECK ============ #
+        if 'TEST_LOG_ACTIONS' in os.environ:
+            logger.warning("Got check result: %d for '%s'",
```

Comment:
Are you only using them into test? The feature of enabling debug base on file is kinda good actually but we may define a scope / use case and then implem in its own Issue. 
Anyway, my question on the debug what more because we have a warning. Is info enough or you think it's already too verbose?